<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperariosPartes</class>
 <widget class="QMainWindow" name="OperariosPartes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>462</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabOperariosPartes">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabOperario">
       <attribute name="title">
        <string>Operarios</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableOperarios">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutOp">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Filtrar por</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbFiltroOp"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Filtro</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edtFiltroOp"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutOp">
          <item>
           <widget class="QPushButton" name="btnNuevaOp">
            <property name="text">
             <string>Nuevo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnModificarOp">
            <property name="text">
             <string>Modificar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEliminarOp">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImprimirOp">
            <property name="text">
             <string>Imprimir</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerOp">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSalirOp">
            <property name="text">
             <string>Salir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabParte">
       <attribute name="title">
        <string>Partes de Trabajo</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="tablePartes">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutPT">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Filtrar por</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbFiltroPT"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Filtro</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edtFiltroPT"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutPT">
          <item>
           <widget class="QPushButton" name="btnNuevaPT">
            <property name="text">
             <string>Nuevo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnModificarPT">
            <property name="text">
             <string>Modificar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEliminarPT">
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImprimirPT">
            <property name="text">
             <string>Imprimir</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerPT">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSalirPT">
            <property name="text">
             <string>Salir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
